### ISSUE TITLE
Collection proposal from BibExtend group

### ISSUE THREAD

**Issue by danbri** [schemaorg/schemaorg#269]
https://www.w3.org/wiki/WebSchemas/Collection

    This proposal addresses two associated but separate needs. Collections 
    of things - artefacts in a museum, books in a library collection, 
    toys in a play-box, images on a website, etc. - are currently 
    only describable in the context of CollectionPage. Created collections 
    are found in many contexts and there is a need to apply the approach 
    more generically. Multi-part works - especially books (eg. Lord of 
    the Rings), but with potentially broader use (the collection of a 
    sculpture plus photographs and textual description of it) - is not 
    currently describable.

**Comment by danbri** [comment]
I've tagged this as 'rough proposal idea' which is maybe unfair, rather better to say that we would need to work out how collections differed from types/classes.

**Comment by danbri** [comment]
Based on a quick twitter exchange, it sounds like this proposal has served its purpose (the creation of isPartOf / hasPart plus ItemList fixes) and won't need to be progressed further. 
- https://twitter.com/rjw/status/560083536968175616 " Was hoping that lists [ordered/non-ordered] would solve it. List + CreativeWorlk could equal Collection"
- https://twitter.com/denials/status/560100371474165760 "Perhaps make it a #schemabibex action item to add 1 or more collection example to our "Recipes and Guidelines"?"

**Comment by danbri** [comment]
@dbs @Dataliberate ... are we ok to close this one? /cc @vholland

**Comment by Dataliberate** [comment]
Not so sure.

Although you could describe a collection as being a CreativeWork and its members using hasPart, however I believe there is need for a CreativeWork subtype specifically to assert it is a Collection.   

This subtype was supposed to be part of the bib extension proposal - the examples made it into the proposal https://github.com/schemaorg/schemaorg/blob/sdo-ganymede/data/ext/bib/bsdo-collection-examples.txt but the definition seems to have been lost from the rdfa - I'll reinsert it.

**Comment by stuartasutton** [comment]
+1

I agree that while "you could describe a collection as being a
CreativeWork", that could be asserted about any of the subtypes of
CreativeWork. Good to see it reinserted, Richard. A specific subtype of
Collection would have clear uses with learning resources where resources
frequently assert membership in collections.

**Comment by danbri** [comment]
Thanks. I've tagged this issue with "extension tracking" so we don't miss it from 'bib:'. Would CreativeWorkCollection or perhaps ItemCollection be ok as a more explicit name for the same idea? ('Collection' is a common enough term for list constucts as well as for real world collections. I'd like to make sure it is not mistaken for http://docs.oracle.com/javase/7/docs/api/java/util/Collection.html or http://www.w3.org/TR/rdf11-mt/#rdf-collections since Collection in that sense is most more like our http://schema.org/ItemList.

Q: Would we use this sense of Collection for a set of otherwise unremarkable secondhand kitchen utensils I am selling via Offer/Product markup?

**Comment by stuartasutton** [comment]
I'm not sure that a bright line can be found between something that is merely a random aggregation (some arbitrary list) with any old thing thrown in and what would be considered a "formal" collection (whatever that is). Is your  "set of otherwise unremarkable secondhand kitchen utensils" an aggregation gathered by some antique collector from across the UK that documents such artifacts from the 17th century? (Maybe that renders them remarkable.) Is a collection of learning resources subject to a common licensing regimen really a collection? Not sure there is a satisfactory answer, Dan, beyond context. In the library world, no one ponders the meaning when the librarians says, "yes we have that in the collection." Or, "that painting is in the National Gallery collection." Are the samples from the Museum of Dirt in Boston really part of "a collection"? (http://www.bestourism.com/items/di/391?title=Museum-of-Dirt-in-Boston-Massachusetts&b=51)

**Comment by danbri** [comment]
@stuartasutton yes, I sympathise with not drawing that line. However we should also be careful to indicate how broadly applicable any new term is, even if it is in an extension. I lean towards CreativeWorkCollection, which doesn't particularly narrow things given that we don't exclude much from being a CreativeWork.

**Comment by Dataliberate** [comment]
That kind of implies that you need a collection of CreativeWorks for it to be a CreativeWorkCollection - which is an obvious use case.

[Creatively] creating a collection of non-creativeWorks, however, is also a valid use case — fossils in a museum, a random collection of kitchen utensils to sell. It is the ‘collecting’ together that is creative.

I agree that drawing that line is difficult for some uses where the creative effort is minimal — was there any creative effort in selecting which kitchen utensils would sell well together?

I think the benefit of broadening out the definition for a simple Collection subtype of CreativeWork to include most types of things in a collection outweighs the downside of it being over engineered for boxes of assorted stuff in a garage sale.

~Richard

On 28 May 2015, at 16:24, Dan Brickley <notifications@github.com<mailto:notifications@github.com>> wrote:

@stuartasuttonhttps://github.com/stuartasutton yes, I sympathise with not drawing that line. However we should also be careful to indicate how broadly applicable any new term is, even if it is in an extension. I lean towards CreativeWorkCollection, which doesn't particularly narrow things given that we don't exclude much from being a CreativeWork.

**Comment by danbri** [comment]
Didn't we did this? https://schema.org/Collection

**Comment by danbri** [comment]
well kind of.

/Collection was originally somewhat implicitly CreativeWork oriented. I made some modest tweak to its wording recently to allow a subtype for /ProductCollection but potentially there could also be a more explicit subtype for creative works, or other more specific subtypes e.g. museums etc. At this stage we are brainstorming unless some service/product is being blocked by lack of type, so I think @RichardWallis was right to tag this for the suggestions-questions-brainstorming repo.
